sdks/java/io/hdfs/pom.xml

```diff
@@ -83,5 +83,14 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- see HDFSFileSourceTest commented block
```

Comment:
I'm not sure what comment block are you referring to. Either way, I'd remove the commented dependency.